modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/actions/jms/client/Send.java

```diff
@@ -88,13 +89,21 @@ public BValue execute(Context context) {
 
         validateParams(bConnector);
 
+        // set return value to the current frame
+        BValue valueRef = new BBoolean(true);
+        context.getControlStackNew().currentFrame.returnValues[0] = valueRef;
+
         //Getting the map of properties.
-        BMap<BString, BString> properties = (BMap<BString, BString>) bConnector.getValue(0);
+        BMap<String, BString> properties = (BMap<String, BString>) bConnector.getValue(0);
 
         //Create property map to send to transport.
-        Map<String, String> propertyMap = properties.keySet()
-                .stream()
-                .collect(Collectors.toMap(BString::stringValue, k -> properties.get(k).stringValue()));
+//        Map<String, String> propertyMap = properties.keySet()
```

Comment:
Better remove this commented out lines.

Reply:
I checked the file. It doesn't exist in ballerina anymore. 
Thanks